src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_client_factory.py

```diff
@@ -42,3 +45,15 @@ def cf_acr_tasks(cli_ctx, *_):
 
 def cf_acr_runs(cli_ctx, *_):
     return get_acr_service_client(cli_ctx).runs
+
+
+def cf_acr_scope_maps(cli_ctx, *_):
+    return get_acr_service_client(cli_ctx).scope_maps
+
+
+def cf_acr_tokens(cli_ctx, *_):
+    return get_acr_service_client(cli_ctx).tokens
+
```

Comment:
`scope_maps` and `tokens` should also use the preview API version ? // @mnltejaswini

Reply:
They have the preview API set by default, so we don't need to specify it. The problem is with the registries client